### PR TITLE
Review one of my past projects

### PR DESCRIPTION
**Pull request by Cabraham1**
The goal of this project is to play the role of a code reviewer

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments. 
+"use strict";
```
Comment (Cabraham1) [inline js/index.js:28]: please try not to leave an unnecessary comment in your codebase as it make your work not looking not clean 😉

File: README.md [hunk 2]
```diff
@@ -1,288 +1,5 @@
...
-
-<!-- LICENSE -->
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Cabraham1) [inline README.md:288]: Please try to edit the ReadMe file as it gives a clear description of your project 😃

**Review by Cabraham1 — COMMENTED** [review]
#Change is required ♻️ 

what a great design, I see you have really put in a lot of work here well done, but you still have some little changes to make as it is a requirement for your project approval. 👍🏿 

#Highlights

Good implementations ✔️

Please check my comments below and make all the required changes

Submit for a re-review when you are done😊

Thank you

File: README.md (hunk 2 above)
Comment (bienvenuushindi) [reply README.md:288]: > Please try to edit the ReadMe file as it gives a clear description of your project smiley

:red_circle: You have well asking for this change. But I think this comment is not clear enough for the reader about what part of the README should be updated. The best way could be to point out the specific section that needs an update.

You  could say: :arrow_down: 
- [ ] Please edit the Authors section of the README. As you can see it still has the placeholders handles Kindly replace the links with your own accounts' links. And as you are the only contributor to the project, kindly remove the second Author2 placeholder.

Briefly for every section that needs an update try to point it out and tell the student what changes you expect from the section.

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
Hi @Cabraham1  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
May I highlight the following:

- Great start :+1: :rocket: 

## Required Changes ♻️
- [ ] Congratulations for the inline comments. But they should appear under the summary comment like this one. The way they appear means that you didn't follow the Github features properly. Your comments  should appear as or form or constitute one block. Please, watch this short [video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d), to learn how to use GitHub inline features as we do here in Microverse. Note: All your comments suppose to be below the summary template. Thanks. :+1: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Cabraham1) [reply README.md:288]: thank you very much for this insight

File: README.md [hunk 3]
```diff
@@ -1,288 +1,5 @@
...
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
-
```
Comment (Cabraham1) [inline README.md:68]: please add a project title as it gives clarity to your project

File: README.md [hunk 4]
```diff
@@ -1,288 +1,5 @@
...
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
```
Comment (Cabraham1) [inline README.md:221]: Please edit the Authors section of the README. As you can see it still has the placeholder handles Kindly replace the links with your own accounts' links. And as you are the only contributor to the project, kindly remove the second Author2 placeholder.

File: README.md [hunk 5]
```diff
@@ -1,288 +1,5 @@
...
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
-
```
Comment (Cabraham1) [inline README.md:231]: Please edit the future section of the README. As you can see it still has the placeholders handles

File: README.md [hunk 6]
```diff
@@ -1,288 +1,5 @@
...
-<!-- CONTRIBUTING -->
-
-## 🤝 Contributing <a name="contributing"></a>
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Cabraham1) [inline README.md:242]: Please edit the Contribution section of the README. As you can see it still has the placeholders handles

File: README.md [hunk 7]
```diff
@@ -1,288 +1,5 @@
...
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
-
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
```
Comment (Cabraham1) [inline README.md:74]: Please edit the Description section of the README. As you can see it still has the placeholder handles

File: README.md [hunk 8]
```diff
@@ -1,288 +1,5 @@
...
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
```
Comment (Cabraham1) [inline README.md:105]: Please edit the Key feature section of the README. As you can see it still has the placeholder handles

Comment (Cabraham1) [inline README.md:105]: Please edit the Key feature section of the README. As you can see it still has the placeholder handles

**Review by Cabraham1 — COMMENTED** [review]
# Changes Required ♻️

#Highlights

Good implementations ✔️

what a great design, I see you have really put in a lot of work here well done, but you still have some little changes to make as it is a requirement for your project approval. 👍🏿

Please check my comments below and make all the required changes


Submit for a re-review when you are done😊

Thank you


@bienvenuushindi

**Review by osamaashraf6 — APPROVED** [review]
Hi @Cabraham1 😊, 

# STATUS: APPROVED! 🥇🟢🟢💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ApprouvéGIF

## Highlights

- [ ] Good communication with the student✔️
- [ ] Polite comments✔️
- [ ] Used at least two inline comments✔️
- [ ] Clear and concise comments ✔️

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**